keps/sig-storage/20190530-pv-health-monitor.md

```diff
@@ -32,7 +32,7 @@ status: implementable
 - [Proposal](#proposal)
 - [Implementation](#implementation)
   - [CSI changes](#csi-changes)
-    - [Add GetVolume RPC](#add-getvolume-rpc)
+    - [Add ControllerGetVolume RPC](#add-controllergetvolume-rpc)
```

Comment:
I think the metadata section above can be removed with the new kep format.

Reply:
Done